server/encoding/yaml/conversion/parser/assertion_parser_test.go

```diff
@@ -364,7 +364,6 @@ func TestParseAssertion(t *testing.T) {
 			require.NoError(t, err)
 			assert.Equal(t, testCase.ExpectedOutput.Attribute, output.Attribute)
 			assert.Equal(t, testCase.ExpectedOutput.Operator, output.Operator)
-			assert.Equal(t, testCase.ExpectedOutput.Value, output.Value)
```

Comment:
This was used when we had a `string` Value. Now, this is irrelevant and we should use assertExpression instead.